guacamole/src/main/webapp/app/login/loginModule.js

```diff
@@ -23,4 +23,4 @@
 /**
  * The module for the login functionality.
  */
-angular.module('login', ['element']);
+angular.module('login', ['element', 'navigation']);
```

Comment:
Then again ... it's hard to argue against having such a line, if it's a legit thing for us to commit ```TODO``` and ```TODONT``` prior to completion.

I guess this is kind of a TODO without the explicit TODO...